server/src/main/java/org/jboss/as/server/parsing/StandaloneXml.java

```diff
@@ -46,14 +48,26 @@
  */
 public class StandaloneXml extends CommonXml {
```

Comment:
I'd like this class to be final now, and CommonXml.parseDeployments should be made protected. That will lock down the API to just the constructors and the XMLElementReader<List<ModelNode>> and XMLElementWriter<ModelMarshallingContext> interfaces.